src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -200,8 +203,12 @@ private bool ValidOptions(ILogger logger)
 
         private TokenResult AccessToken(IPublicClientAuth publicClientAuth, CommandExecuteEventData eventData)
         {
+            var authParams = new AuthParameters(
+                AzureAuth.Ado.Constants.Client.VisualStudio,
+                this.Tenant,
+                new[] { AzureAuth.Ado.Constants.Scope.AzureDevOpsDefault });

```

Comment:
Do we not need custom scope to pass through here?

Reply:
Confusingly, that's a different scope. Azure AD scopes are not the same as PAT scopes for Azure DevOps. In this case it's the former and that's the only one we need in this case, so there's no reason to do pass through.